src/pyenphase/updaters/meters.py

```diff
@@ -111,7 +111,10 @@ async def probe(
                     self.production_meter_type = meter["measurementType"]
                     # save meter identifier for link between /ivp/meters and /ivp/meters/readings
                     self.production_meter_eid = meter["eid"]
-                else:
+                if meter["measurementType"] in [
+                    CtType.NET_CONSUMPTION,
+                    CtType.TOTAL_CONSUMPTION,
+                ]:
```

Comment:
```suggestion
                if meter["measurementType"] in (
                    CtType.NET_CONSUMPTION,
                    CtType.TOTAL_CONSUMPTION,
                ):
```

tuples are cheaper than lists

Reply:
Done, thanks